File: BriefAssistant/Models/ContactInfo.cs
using System.ComponentModel.DataAnnotations;
using System.Runtime.Serialization;

namespace BriefAssistant.Models
{
    [DataContract(Namespace = "")]
    public class ContactInfo
    {
        [DataMember]
        public string Name { get; set; }
        [DataMember]
        public Address Address { get; set; }

        [DataMember]
        public string Email { get; set; }

        [DataMember]
        public string Phone { get; set; }

        [DataMember]
        public string BarId { get; set; }
    }
}

File: BriefAssistant/ClientApp/src/app/briefs/shared/ContactInfo.ts
import { Address } from "./Address";

export class ContactInfo {
  phone: string;
  email: string;
  address: Address;
  name: string;
  barId: string;

  constructor() {
    this.phone = "";
    this.email = "";
    this.address = new Address();
    this.name = "";
    this.barId = "";
  }
}
